src/Components/PaperEditor/ProgrammeScriptContainer/ExportDropdown/edl.js

```diff
@@ -27,17 +27,34 @@ const formatToEDLEvent = (transcript, element) => {
   return result;
 };
 
-const getEDLSq = (title, elements, transcripts) => {
-  return elements.reduce((res, element) => {
-    if (element.type === 'paper-cut') {
-      const transcript = getCurrentTranscript(element, transcripts);
-      const edlEvent = formatToEDLEvent(transcript, element);
-      res.index += 1;
-      edlEvent.id = res.index;
-      res.events.push(edlEvent);
-      console.log('res', element, transcript);
+const mergeConsecutiveElements = (elements) => {
+  const paperCuts = elements.filter(element => element.type === 'paper-cut');
+
+  return paperCuts.reduce((accumulator, currentElement) => {
+    const prevElement = accumulator[accumulator.length - 1];
```

Comment:
I think the problem is here - if the accumulator is empty then prevElement is ???
I'd change this reduce function body to something like: 
```
    if (accumulator.length > 0) {
      const prevElement = accumulator[accumulator.length - 1];
      const areElementsConsecutive = currentElement.transcriptId === prevElement.transcriptId && prevElement.end === currentElement.start;
      if (areElementsConsecutive) {
        prevElement.end = currentElement.end;
      }
    }
    else {
      accumulator.push(currentElement);
    }

    return accumulator;
```
However!
This code is mutating objects within arrays which are being passed all around this module - lots of potential for things to get messed up, so better to write it immutably, such as:
```
    if (accumulator.length > 0) {
      const prevElement = accumulator[accumulator.length - 1];
      const areElementsConsecutive = currentElement.transcriptId === prevElement.transcriptId && prevElement.end === currentElement.start;
      if (areElementsConsecutive) {
        const mergedElement = { ...prevElement, end: currentElement.end };

        return [ ...accumulator.slice(0, accumulator.length - 1), mergedElement ];
      }
    } else {
      return [ ...accumulator, currentElement ];
    }

Reply:
If I add three paragraphs individually or together to a script, the EDL lists them as one clip (I hope that was the right test?)